ui/sortable.js

```diff
@@ -788,12 +788,10 @@ return $.widget("ui.sortable", $.ui.mouse, {
 							.addClass(className || that.currentItem[0].className+" ui-sortable-placeholder")
 							.removeClass("ui-sortable-helper");
 
-					if ( nodeName === "tr" ) {
-						that.currentItem.children().each(function() {
-							$( "<td>&#160;</td>", that.document[0] )
-								.attr( "colspan", $( this ).attr( "colspan" ) || 1 )
-								.appendTo( element );
-						});
+					if ( nodeName === "tbody" ) {
+						that._createTrPlaceholder( that.currentItem.find( "tr:first" ), $( "<tr></tr>", that.document[0] ).appendTo( element ) );
```

Comment:
* Avoid use of `:first` selector, use a filter method like `.eq( 0 )`.
* Empty element construction uses just the opening name: $( "<tr>" )
* Missing spaces inside brackets.
* This line exceeds 100 characters.